images/win/scripts/Installers/Validate-ToolCache.ps1

```diff
@@ -12,6 +12,21 @@ function GetChildFolders {
     return Get-ChildItem -Path $Path -Directory -Name
```

Comment:
We can add try-catch here to produce a human-readable error if the path doesn't exist

Reply:
I believe it's unnecessarily, because, before first call of Get-ChildItem function, we've got several checks that throw human-readable error in case if path or part of path is missing.